Content.Server/Temperature/Components/TemperatureComponent.cs

```diff
@@ -39,33 +54,10 @@ public class TemperatureComponent : Component
             }
         }
 
-        [ViewVariables] public float SpecificHeat => _specificHeat;
-
-        [DataField("heatDamageThreshold")]
-        private float _heatDamageThreshold = default;
-        [DataField("coldDamageThreshold")]
-        private float _coldDamageThreshold = default;
-        [DataField("tempDamageCoefficient")]
-        private float _tempDamageCoefficient = 1;
-        [DataField("currentTemperature")]
-        private float _currentTemperature = Atmospherics.T20C;
-        [DataField("specificHeat")]
-        private float _specificHeat = Atmospherics.MinimumHeatCapacity;
-
         public void Update()
         {
             var tempDamage = 0;
-            DamageType? damageType = null;
-            if (CurrentTemperature >= _heatDamageThreshold)
-            {
-                tempDamage = (int) Math.Floor((CurrentTemperature - _heatDamageThreshold) * _tempDamageCoefficient);
-                damageType = DamageType.Heat;
-            }
-            else if (CurrentTemperature <= _coldDamageThreshold)
-            {
-                tempDamage = (int) Math.Floor((_coldDamageThreshold - CurrentTemperature) * _tempDamageCoefficient);
-                damageType = DamageType.Cold;
-            }
+            DamageTypePrototype? damageType = null;
```

Comment:
These two variables can be moved down if you are moving the rest of the damage dealing code down, they are not needed to be declared this early.